crates/rspack_plugin_css/src/visitors/mod.rs

```diff
@@ -27,6 +27,7 @@ pub fn analyze_dependencies(
     deps: Vec::new(),
     code_generation_dependencies,
     diagnostics,
+    in_support_contdition: false,
```

Comment:
`@supports` is not supported in rspack, is there any specific case for adding this?

Reply:
you can comment `in @supports url` case for now, and uncomment it after we support `@supports`